index.js

```diff
@@ -3,7 +3,8 @@ var settings = {
         host: process.env.MQTT_HOST || '',
         user: process.env.MQTT_USER || '',
         password: process.env.MQTT_PASS || '',
-        clientId: process.env.MQTT_CLIENT_ID || null
+        clientId: process.env.MQTT_CLIENT_ID || null,
+        protocol: process.env.MQTT_PROTOCOL || 'mqtt'
```

Comment:
Do we really need to set protocol. I usually set it using host like mqtts://mqtt.com:123456. Also I don't like to have mqtt as default, this might be a security bridge. I think mqtts is better.

Reply:
You need to set the protocol if the host is anything but null. Otherwise the mqtt library throws an exception.

Default to mqtts if you prefer.